DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: jenner
 Title: Internal Montagu Helpers
-Version: 0.0.6
+Version: 0.0.7
```

Comment:
You've already created 0.0.7 as part of #7 - this should be 0.0.8 (see https://github.com/vimc/jenner/blame/master/DESCRIPTION)

Reply:
Yes. You are right. We added the create_touchstone function for 0.0.7. Corrected now - here and the NEWs.